spec/mailers/shipment_mailer_spec.rb

```diff
@@ -44,4 +58,21 @@
     shipment_email = Spree::ShipmentMailer.shipped_email(shipment)
     expect(shipment_email.body).to include("Your order from #{distributor.name} has been shipped")
   end
+
+  it "picked_up_email accepts an shipment id as an alternative to an Shipment object" do
+    expect(Spree::Shipment).to receive(:find).with(shipment.id).and_return(shipment)
+    expect {
+      Spree::ShipmentMailer.picked_up_email(shipment.id).deliver_now
+    }.to_not raise_error
+  end
+
+  it "includes the distributor's name in the subject" do
+    picked_up_email = Spree::ShipmentMailer.picked_up_email(shipment)
+    expect(picked_up_email.subject).to include("#{distributor.name} Shipment Notification")
+  end
+
+  it "includes the distributor's name in the body" do
+    picked_up_email = Spree::ShipmentMailer.picked_up_email(shipment)
+    expect(picked_up_email.body).to include("Your order from #{distributor.name} has been picked-up")
```

Comment:
line is too long here ;)

Reply:
ah didn't see this. Thanks for telling me!

Reply:
You can check locally:
```
bundle exec rubocop
```